src/screens/Messages/Tabs/TeamChat.js

```diff
@@ -69,16 +66,13 @@ class TeamChat extends Component {
     navigateToChat(mockConversation, room) {
         this.props.navigation.navigate('ConversationScreen', {
             conversation: mockConversation,
-            room: room,
-            userID: this.state.userID,
-            getChatRooms: this.getChatRooms
+            room: room
         })
     }
 
     removeTeam = async (room) => {
 
         let token = await AsyncStorage.getItem("token")
-        let user = await AsyncStorage.getItem("userID")
```

Comment:
can you add [userID ](https://github.com/Airbenders-490/AirbendersFE/blob/bdaaa94673ad64e7d8758a74479a50e394b9757e/src/screens/Messages/Tabs/TeamChat.js#L81) back please